### PR TITLE
Multiple updates

### PR DESCRIPTION
**Pull request by sssemil**
Fix configuration stuff
Use not deprecated activity
Code cleanup
Fix travis

### CODE REVIEW

File: src/main/res/values/donottranslate.xml [hunk 1]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <string name="setting_verbose_enabled_key">setting_verbose_enabled_key</string>
```
Comment (benhylau) [inline src/main/res/values/donottranslate.xml]: Why move these to **strings.xml**? If we translate to different languages, these don't need to be translated, that's why I prefer to keep them in a separate file.

Comment (sssemil) [inline src/main/res/values/donottranslate.xml]: I've added translatable=false, that seems to be a righter way

Comment (benhylau) [inline src/main/res/values/donottranslate.xml]: Sounds good :+1:

**Comment by benhylau** [comment]
@sssemil Sorry I didn't get to these PRs earlier. Just have time to get back to this project now. Thanks for contributing!

**Pull request merged**